fec.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
         exposes: {
             './RootApp': resolve(__dirname, './src/AppEntry'),
             './SystemDetail': resolve(__dirname, './src/index.js'),
-            './OsExposureReport': resolve(__dirname, './src/Components/SmartComponents/Reports/OsExposureReport')
+            './OsExposureReport': resolve(__dirname, './src/Components/SmartComponents/Reports/OsExposureReport/OsExposureReport')
```

Comment:
I am not sure why I am getting a blank PDF when trying to create some report. Probably, we are missing .js:

```
            './OsExposureReport': resolve(__dirname, './src/Components/SmartComponents/Reports/OsExposureReport/OsExposureReport.js')
```

But perhaps it's because I have tried to test without spinning the PDF service locally.

Reply:
yeah, you have to run them together locally.